.editorconfig

```diff
@@ -0,0 +1,24 @@
+root = true
+
+[*]
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{cpp,h,py,sh,yml}]
+indent_style = space
+indent_size = 2
+max_line_length = 140
+
+[*.py]
+indent_size = 4
```

Comment:
I did that because for most things python files use the same settings as everything else, except for that it uses an indent size of 4 instead of 2. If you think that python files should use an indent size of 2, I can remove that bottom one and then update all the python scripts to use that indent size.

Reply:
Most of the python files come from elsewhere and shouldn't be touched. The only one that's "ours" as far as I know is src/displayapp/fonts/generate.py. It's using 4 spaces, but maybe it should rather be consistent across the whole project. It could be matched by checking in the src directory only.